### ISSUE TITLE
ArcticDEM updates

### ISSUE THREAD

**Issue by trey-stafford** [nsidc/qgreenland#797]

**Comment by trey-stafford** [comment]
It does not look like we currently have any documentation on addons / ArcticDEM in RTD. 

The existing/old website has this text on the downloads page, under Additional Data for QGreenland":

> [32-meter resolution ArcticDEM](http://data.pgc.umn.edu/gis/packages/qgreenland/packages/addons/) (digital elevation model): The Polar Geospatial Center has provided a custom Greenland 32-m resolution ArcticDEM for added detail. Download this add-on package and place it in a suitable location in your filesystem. Unzip the package and then follow the directions in the 'README.txt' file included in the package.


On the new website, on the "More Data" page, under "Custom QGreenland Resources":

> [32-meter resolution ArcticDEM](http://data.pgc.umn.edu/gis/packages/qgreenland/packages/addons/) (digital elevation model): The Polar Geospatial Center has provided a custom Greenland 32-m resolution ArcticDEM for added detail. Download this add-on package and place it in a suitable location in your filesystem. Unzip the package and then follow the directions in the 'README.txt' file included.

**Comment by trey-stafford** [comment]
PGC provided us with a link to an ArcGIS image server: https://overlord.pgc.umn.edu/arcgis/rest/services/elevation/pgc_arcticdem_mosaics_latest/ImageServer

This link can be used directly in QGIS's "ArcGIS REST Server" data source manager. It adds a WMS-like grayscale image with a colorbar/legend (without units).  QGIS doesn't appear to allow the option to change the symbology. 

A WMS endpoint is also available: https://overlord.pgc.umn.edu/arcgis/services/elevation/pgc_arcticdem_mosaics_latest/ImageServer/WMSServer?request=GetCapabilities&service=WMS

Using the WMS interface of the data source manager, I can add this URL and QGIS recognizes a layer that can be added. Note that the "PNG" image encoding option must be selected for the layer to work. This experience is similar to the ArcGIS REST Server option - a legend is provided for a grayscale image.

Finally, WCS is a listed option: https://overlord.pgc.umn.edu/arcgis/services/elevation/pgc_arcticdem_mosaics_latest/ImageServer/WCSServer

Connecting to the WCS service in QGIS shows an available layer named "1". Attempting to add this layer results in an error/warning icon next to the layer indicating that the data source is unavailable.

**Comment by MattF-NSIDC** [comment]
>If we bundle the style with the data, we could look at adding it as a .qlr file and include additional metadata like the abstract/citation info as we normally do for QGreenland layers...

Exploring what can be done with QLR files feels like it will be valuable research for QGreenland-Net.

**Comment by trey-stafford** [comment]
Some replacement text for the website: 

```
32-meter resolution ArcticDEM (digital elevation model): The Polar Geospatial
Center has provided a custom Greenland 32-m resolution ArcticDEM for added
detail. Two download packages are provided for the v4.1 ArcitcDEM:

* arcticdem_v4.1_32m_greenland_dem_only.zip: A ~4.1GB package containing only
  the 32-m resolution DEM.

* arcticdem_v4.1_32m_greenland_dem_full.zip: A ~9.5GB package containing the
  32-m resolution DEM and ancillary datasets (e.g., a "count" dataset giving the
  number of contributing DEMs for each pixel).

Download one of these add-on packages and place it in a suitable location in your
filesystem. Unzip the package and then follow the directions in the 'README.txt'
file included.
```

**Comment by trey-stafford** [comment]
> Exploring what can be done with QLR files feels like it will be valuable research for QGreenland-Net.

Agreed. I tried exporting the layer w/ a manually edited abstract as a `.qlr`. That seemed to work OK! Still need to do some more testing to ensure we can e.g.., save w/ a relative path.

For now though, I think I'll update the "Styling" section of the package's README to point to QGIS resources:

```
-------------------
Styling this data:
-------------------

For information on changing the symbology of raster datasets in QGIS, see:

* QGIS lesson on changing raster symbology:
  https://docs.qgis.org/3.28/en/docs/training_manual/rasters/changing_symbology.html

* QGIS documentation on raster symbology properties:
  https://docs.qgis.org/3.28/en/docs/user_manual/working_with_raster/raster_properties.html#symbology-properties
```

**Comment by trey-stafford** [comment]
PR (draft) that leverages the ArcGIS image server to provide an online-access layer: https://github.com/nsidc/qgreenland/pull/799

**Comment by trey-stafford** [comment]
The PGC also provided us with some VRT files that reference data on AWS. I was not very successful with adding these to QGIS. Adding them as a layer causes QGIS to freeze. 

#799 looks like our best option atm to add ArcticDEM as an online-access layer.
